powerauth-webauth/src/main/java/io/getlime/security/powerauth/app/webauth/configuration/PowerAuthWebServiceConfiguration.java

```diff
@@ -60,14 +70,55 @@ public PowerAuthServiceClient powerAuthClient(Jaxb2Marshaller marshaller) {
             ClientInterceptor interceptor = securityInterceptor();
             client.setInterceptors(new ClientInterceptor[] { interceptor });
         }
+        // whether invalid SSL certificates should be accepted
+        if (acceptInvalidSslCertificate) {
+            trustAllCertificates();
+        }
         return client;
     }
 
     @Bean
     public PushServerClient pushServerClient() {
         PushServerClient client = new PushServerClient();
         client.setServiceBaseUrl(powerAuthPushServiceUrl);
+        // whether invalid SSL certificates should be accepted
+        if (acceptInvalidSslCertificate) {
+            trustAllCertificates();
+        }
         return client;
     }
 
+    /**
+     * Activate trust in all SSL certificates including invalid ones for non-production use.
+     */
+    private void trustAllCertificates() {
```

Comment:
It is not the same project - one commit affects powerauth-webauth.war and second commit affects powerauth-webauth-client.war. I thought we needed to disable SSL certificate check in each war file separately? If not, then I will change it.